consensus/ibft/src/integration-test/java/org/hyperledger/besu/consensus/ibft/support/TestContextBuilder.java

```diff
@@ -273,28 +273,28 @@ private static ControllerAndState createControllerAndFinalState(
     final StubGenesisConfigOptions genesisConfigOptions = new StubGenesisConfigOptions();
     genesisConfigOptions.byzantiumBlock(0);
 
-    final ProtocolSchedule protocolSchedule = IbftProtocolSchedule.create(genesisConfigOptions);
+    final ProtocolSchedule protocolSchedule = BftProtocolSchedule.create(genesisConfigOptions);
 
     /////////////////////////////////////////////////////////////////////////////////////
     // From here down is BASICALLY taken from IbftBesuController
```

Comment:
nit: doco reference need to be updated.

Reply:
this is still valid - this class represents the IBFT-BesuController - i.e. this class is still specific to the consensus mechanism.